server/version/BUILD

```
@@ -1,15 +1,35 @@
 load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
+genrule(
+    name = "populate_version",
+    outs = ["version.txt"],
+    cmd_bash = """
+        version=$$(grep ^STABLE_VERSION_TAG bazel-out/stable-status.txt | cut -d' ' -f2); \
+        echo "$$version" > $@;
```

nit: to avoid a newline after the version, do:

```suggestion
        printf '%s' $$version > $@;
```